src/EditorFeatures/CSharpTest/Formatting/Indentation/SmartIndenterTests_Performance.cs

```diff
@@ -2211,7 +2211,7 @@ static void Main()
 
             AssertSmartIndent(
                 code,
-                expectedIndentation: 4);
+                expectedIndentation: 12);
```

Comment:
this result is more appropriate.  inside an inaction region we now switch to block-indentation, picking the indentation after you hit enter based on the indentation of the previous line.  this seems sane and sensible